staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/conversion.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/apiserver/pkg/apis/audit"
+)
+
+// Convert_audit_GroupResources_To_v1alpha1_GroupResources is a manually created conversion
```

Comment:
Manual conversion added. Tried to copy the RBAC conversion https://github.com/kubernetes/kubernetes/blob/v1.7.4/pkg/apis/rbac/v1alpha1/conversion.go